qemu/tests/virtio_port_hotplug.py

```diff
@@ -71,8 +74,16 @@ def run(test, params, env):
                 error_context.context("Load  module %s" % module, logging.info)
                 session.cmd("modprobe %s" % module)
         session.close()
+    check_pid_cmd = 'ps aux | grep "%s"| grep -v "grep"'
```

Comment:
`kill -9 `pgrep -f host_script``   or `pkill -f host_script`

Reply:
Thanks and update.